Remotion/Web/Core/ContentSecurityPolicy/INonceGenerator.cs

```diff
@@ -0,0 +1,29 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+#if !NETFRAMEWORK
+using System;
+
+namespace Remotion.Web.ContentSecurityPolicy
+{
+  /// <summary>
+  /// Represents an API for generating alpha numeric nonce values.
+  /// </summary>
+  public interface INonceGenerator
```

Comment:
please add a threadsafety xmldoc to the itnerface. see other occurances in the code. we want to document this interface should always be implemented threadsafe.
please add a seealso xmldoc (see other occurances) and reference the implementation. We do this to make it visually easy to find the implementations (at least when they're located in the same assembly)